vllm/engine/async_llm_engine.py

```diff
@@ -510,6 +511,12 @@ def __init__(self,
         self.log_requests = log_requests
         self.engine = self._init_engine(*args, **kwargs)
 
+        if self.engine_use_ray:
```

Comment:
we can hard fail here, and put an env var to turn it on  with a warning.

something like:

```python
if self.engine_use_ray:
    if envs.VLLM_ALLOW_ENGINE_USE_RAY:
        # print warning
    else:
        # directly error, with the error message pointing to the RFC issue
```

for the link, please use full url.

Reply:
Thank you for your comment @youkaichao. I added modifications to address your suggestions.